Pokedex_ios/View/Controller/PokemonListViewController.swift

```diff
@@ -9,28 +9,32 @@ import UIKit
 
 class PokemonListViewController: UIViewController {
     
-    @IBOutlet weak var tableView: UITableView!
+    @IBOutlet weak var pokemonsTableView: UITableView!
+    
+    let pokeImageHeader = "PokeballSVG"
```

Comment:
Add this constants to an external constants file, not here

Reply:
but it could be used more as is a common background on the app. Thats why a global constant suits better for this.